web/app.php

```diff
@@ -2,7 +2,7 @@
 
 use Symfony\Component\HttpFoundation\Request;
 
-/**
+/*
```

Comment:
Yeah, StyleCI believes `$var` is a structural element and thus it cannot have docblocks. However, I think it's still very usefull for ones understanding to have the docblocks here.

What about disabling the [phpdoc_to_comment](https://styleci.readme.io/docs/fixers#phpdoc_to_comment) fixer in the styleci config?